.github/workflows/windows_ROS2.yaml

```diff
@@ -0,0 +1,43 @@
+name: CI
+on:
+  pull_request:
+  push:
+    branches:
+      - foxy-devel
+  schedule:
+    # every Monday
+    - cron:  '0 0 * * 1'
+
+jobs:
+  build:
+    runs-on: [windows-latest]
+    strategy:
+      fail-fast: false
+      matrix:
+        ROSDISTRO: [foxy]
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+        path: src
+    - name: Install
+      shell: cmd
+      run: |
+        choco sources add -n=roswin -s https://aka.ms/ros/public --priority 1
```

Comment:
I'd prefer this use the github action. However, we can pull this now to close this out.